rootfs/api/tests/test_pods.py

```diff
@@ -415,6 +415,8 @@ def test_scale_errors(self, mock_requests):
         body = {'web': -1}
         response = self.client.post(url, body)
         self.assertEqual(response.status_code, 400, response.data)
+        self.assertEqual(response.data, {'detail': "Invalid scaling format: "
+                                         "['Must be greater than or equal to zero']"})
```

Comment:
Having a value with datatypes `Invalid scaling format: ['Must be greater than or equal to zero']` is a little tricky to parse, and also doesn't really follow standard json usage. Something like this makes the error much easier to work with and is a lot less confusing for people trying to understand it.

```json
{
    "detail": {
        "scaling_format": ["Must be greater than or equal to zero"]
    }
}
```